### DISCUSSION TITLE
如何根据文章发布时间获取最新的文章列表

### DISCUSSION THREAD

**Discussion by WAGFS** [halo-dev/halo#6796] [Q&A] [answered]
在使用/apis/api.content.halo.run/v1alpha1/posts时，发现默认获取到的文章列表是根据创建时间来的，我想要根据发布时间来获取文章，但是不知道如何传递字段，希望日后文档中可以提供一些更详细的参数示例或者说明，感谢~
![image](https://github.com/user-attachments/assets/786ad320-cc26-4511-b6fe-3b0c4be610a9)

**Comment by ruibaby** [comment, marked as answer]
try `/apis/api.content.halo.run/v1alpha1/posts?sort=spec.publishTime,desc`

其中 sort 为数组，可以传入多个，比如 `/apis/api.content.halo.run/v1alpha1/posts?sort=spec.publishTime,desc&sort=spec.priority,desc`

**Reply by WAGFS** [reply]
谢谢，它确实有用🚀

**Reply by CpythonJava** [reply]
It works!Thank you!
